brokerapi/brokers/cloudsql/broker.go

```diff
@@ -95,28 +106,30 @@ func (b *CloudSQLBroker) Provision(instanceId string, details models.ProvisionDe
 		return models.ServiceInstanceDetails{}, fmt.Errorf("Error unmarshalling plan features: %s", err)
 	}
 
-	// set default parameters or cast strings to proper values
-	firstGenTiers := []string{"d0", "d1", "d2", "d4", "d8", "d16", "d32"}
+	var binlogEnabled = false
 	isFirstGen := false
-	for _, a := range firstGenTiers {
-		if a == strings.ToLower(planDetails["tier"]) {
-			isFirstGen = true
-		}
-	}
+	// set default parameters or cast strings to proper values
+	if isPostgres {
+		params["version"] = "POSTGRES_9_6"
```

Comment:
should this be a configuration in the plan or as part of creation? presumably this will change frequently

Reply:
it's consistent to leave it as a provision param. Also, version doesn't affect pricing, so it doesn't seem fitting to make it a plan property. I should set it as a default rather than hardcoding though.